action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
   text:
     description: "The message text to post"
     required: true
+  unfurl_links:
+    description: "Whether links in the message should be unfurled"
+    type: boolean
+    default: true
```

Comment:
Does this default to true in Slack API if not provided?

Reply:
Their docs don't actually say what it defaults to, but my anecdotal testing seems to indicate yes